data/campaigns/Secrets_of_the_Ancients/gui/zombie_recruit_dialog.cfg

```diff
@@ -179,7 +179,7 @@
                 horizontal_alignment = "left"
                 [label]
                     definition = "default_small"
-                    label = "   undead"
```

Comment:
These should probably use the translations from the `wesnoth-help` domain, (https://wiki.wesnoth.org/GettextForWesnothDevelopers#Reusing_mainline_translations).

Reply:
And if the extra leading whitespace is required, then it should be nontranslatable ie `"  " + _"undead"`.